api/config/config_local.json

```diff
@@ -105,7 +105,9 @@
     "enableEventDateModifier": false
   },
   "actionAudit": {
-    "logName": "workbench-action-audit-local"
+    "logName": "workbench-action-audit-local",
```

Comment:
There's some duplication here, but it's because I can't guarantee we won't need to rename a table separately from the dataset or log. For example, adding a column to the JSON in the log means we technically need a new BQ schema for the table.